source/crud/tests/write/findOneAndReplace-upsert_pre_2.6.yml

```diff
@@ -0,0 +1,88 @@
+data:
+    - {_id: 1, x: 11}
+    - {_id: 2, x: 22}
+    - {_id: 3, x: 33}
+# This file includes the same test cases as findOneAndReplace-upsert.yml with
+# some omissions for pre-2.6 servers. We cannot verify the ID of an upserted
+# document in some cases due to SERVER-5289.
+maxServerVersion: '2.4.99'
```

Comment:
Using "X.Y" for `minServerVersion` isn't really an issue, as we can assume the minor version is zero. That doesn't work as nicely for `maxServerVersion`, so we can either use the next minor version or a sufficiently high patch version in the same "X.Y" series. The [CRUD test README](https://github.com/mongodb/specifications/blob/master/source/crud/tests/README.rst) does not specify whether `minServerVersion` and `maxServerVersion` are inclusive or exclusive, so I opted to use a high patch version instead of "2.5" in this case.

On the inclusive/exclusive front, I suppose that's only an edge case when comparing a test's `minServerVersion` against a server with a "0" patch version (e.g. "2.6.0"). To that end, we should probably document that these ranges are inclusive. I opened [SPEC-834](https://jira.mongodb.org/browse/SPEC-834) to track this.